web/html/src/manager/content-management/shared/components/panels/sources/channels/channels-selection.js

```diff
@@ -95,6 +95,20 @@ const ChannelsSelection = (props: PropsType) => {
             options={orderedBaseChannels}
             getOptionLabel={option => option.name}
             getOptionValue={option => option.id}
+            menuPortalTarget={document.body}
+            classNamePrefix={`class-selectedBaseChannel`}
+            styles={
```

Comment:
I know this was here already previously, but would it be possible to switch this over to use the combobox or some other shared input component? Otherwise we'll have to keep updating these config values across two unrelated places.

Reply:
I suppose so... never having touched that piece of code I didn't want to break it now ;)

Reply:
Okay, I wasn't sure whether this change set was a followup to the previous work on this same front or not. In that case could you please make a ticket for this?

Reply:
Sure. But I'll make a ticket to kick off the direct uses of `react-select` rather than just this one

Reply:
There you go: https://github.com/uyuni-project/uyuni/issues/3685

Reply:
Nice, tyvm, I appreciate it. 🍻